test/v1/controllers/requestParsers/validators/AmendBenefitAmountsValidatorSpec.scala

```diff
@@ -88,6 +90,11 @@ class AmendBenefitAmountsValidatorSpec extends UnitSpec with ValueFormatErrorMes
 
     MockedAppConfig.minimumPermittedTaxYear
       .returns(2021)
+
+    MockedAppConfig.featureSwitch.returns(Some(Configuration(ConfigFactory.parseString(
+      """
+        |taxYearNotEndedRule.enabled = true
+      """.stripMargin))))
```

Comment:
I don't think this mock is required here. `amounts` endpoints don't use this validation.